static/app/components/search/sources/apiSource.tsx

```diff
@@ -272,11 +272,11 @@ async function createEventIdLookupResult(
 
 type Props = WithRouterProps<{orgId: string}> & {
   children: (props: ChildProps) => React.ReactElement;
-  organization: Organization;
   /**
    * search term
    */
   query: string;
+  organization?: Organization;
```

Comment:
There is logic in this component for if `organization` doesn't exist (and a test for it) so I made it optional